### PR TITLE
wd_demo.au3 - Au3Check - compatibility

### PR DESCRIPTION
**Pull request by mlipok**

### CODE REVIEW

**Comment by mlipok** [comment]
just added:
_WD_Demo()
moved some declaration to `Global`, some was not strictly defined by `Local` some was not used so I use `#forceref`

File: wd_demo.au3 [hunk 1]
```diff
@@ -5,18 +5,19 @@
...
-Local Const $sElementSelector = "//input[@name='q']"
+#Region - Global's declarations
 
-Local $sDesiredCapabilities, $iIndex, $sSession
-Local $nMsg, $bProcess = False
-
-Local $aBrowsers[][2] = _
-		[["Firefox", SetupGecko], _
+Global Const $sElementSelector = "//input[@name='q']"
+Global Const $aBrowsers[][2] = _
```
Comment (Danp2) [inline wd_demo.au3:11]: These globals could be moved to locals of new _WD_Demo function

**Review by Danp2 — APPROVED** [review]
See comment re: globals, which isn't a big deal.

File: wd_demo.au3 (hunk 1 above)
Comment (mlipok) [reply wd_demo.au3:11]: checking

Comment (mlipok) [reply wd_demo.au3:11]: $sElementSelector is used in `DemoElements()` and `DemoActions()` so I would'nt want to move them.
What is your opionion ?

$aBrowsers is used only in `_WD_Demo()` but I want to split them to 2 spearate function and do not exit main look in another way then using $GUI_EVENT_CLOSE.

Comment (Danp2) [reply wd_demo.au3:11]: Right... I was referring to the globals below $sElementSelector. 

> $aBrowsers is used only in _WD_Demo() but I want to split them to 2 spearate function and do not exit main loop in another way then using $GUI_EVENT_CLOSE.

Not sure what you mean here. Please explain.

Comment (mlipok) [reply wd_demo.au3:11]: For this moment I porpose that you accept this PR as it is.
Then I do my next PR which will show to you exactly what I mean.

**Review by Danp2 — COMMENTED** [review]
I don't believe that you need to merge the changes from the master branch into your patch branch. Thoughts?

**Comment by mlipok** [comment]
It was just that easier for me.

**Pull request merged**